packages/client/src/__tests__/benchmarks/query-performance/caching.bench.ts

```
@@ -0,0 +1,201 @@
+import { withCodSpeed } from '@codspeed/benchmark.js-plugin'
+import { dmmfToRuntimeDataModel, type QueryCompiler, type QueryCompilerConstructor } from '@prisma/client-common'
+import { getDMMF } from '@prisma/client-generator-js'
+import type { JsonQuery } from '@prisma/json-protocol'
+import { ParamGraph } from '@prisma/param-graph'
+import { buildParamGraph } from '@prisma/param-graph-builder'
+import Benchmark from 'benchmark'
+
+import { parameterizeQuery } from '../../../runtime/core/engines/client/parameterization/parameterize'
+import { loadQueryCompiler } from './qc-loader'
+
+let QueryCompiler: QueryCompilerConstructor
```

Biome (2.3.13)

[error] Shouldn't redeclare 'QueryCompiler'. Consider to delete it or rename it.

'QueryCompiler' is defined here:


(lint/suspicious/noRedeclare)